### ISSUE TITLE
il-unit-wordmark (v2) accessibility - touch targets do not have sufficient size or spacing

### ISSUE THREAD

**Issue by dandalpiaz** [web-illinois/toolkit#541]
I hadn't seen this flagged before so I'm guessing that axe-core had a recent update with more WCAG 2.2 tests. The `<p class="il-primary-unit">` and `<h1>` elements within the `<il-unit-wordmark>` component are being flagged for insufficient spacing when both are present. 

Example usage: https://webtheme.illinois.edu/
More info: https://dequeuniversity.com/rules/axe/4.9/target-size
